.github/workflows/javascript_builds.yml

```diff
@@ -4,9 +4,9 @@ on: [push, pull_request]
 # Global Settings
 env:
   GODOT_BASE_BRANCH: master
-  SCONSFLAGS: platform=javascript verbose=yes warnings=extra werror=yes debug_symbols=no --jobs=2
+  SCONSFLAGS: platform=javascript verbose=yes warnings=extra werror=yes debug_symbols=no module_webxr_enabled=no threads_enabled=yes --jobs=2
   SCONS_CACHE_LIMIT: 4096
-  EM_VERSION: 1.39.20
+  EM_VERSION: 2.0.15
```

Comment:
I just updated my containers to 2.0.25 (will be used for 3.4 and 4.0, unless we need to rollback) so we could use it here too.